### PR TITLE
[9.x] Add AWS S3 delete multiple files using deleteObjects method

### PR DESCRIPTION
**Pull request by aneeskhan47**
Hi,

I needed to delete 100+ files on AWS s3. the `Storage::disk('s3')->delete()` delete a single file so if i loop through the 100+ files and do the `Storage::disk('s3')->delete($file)` it will do 100+ request to s3 for 100+ files.

So I searched and I find out that AWS s3 allows you to delete multiple files using the AWS client `deleteObjects` method which accepts an array of object paths e.g `['path/to/file1', 'path/to/file2']`. using this it just took 2 secs to delete 100+ files. 

more info: https://docs.aws.amazon.com/AmazonS3/latest/userguide/example_s3_DeleteObjects_section.html

I first implemented this as a macro on the Storage facade in my projects. but I think this should be in laravel by default :)

Thanks,
Anees

### CODE REVIEW

**Comment by ankurk91** [comment]
I think this should added here instead
https://github.com/thephpleague/flysystem-aws-s3-v3

File: src/Illuminate/Filesystem/FilesystemAdapter.php [hunk 1]
```diff
@@ -110,7 +111,8 @@ public function assertExists($path, $content = null)
 
         foreach ($paths as $path) {
             PHPUnit::assertTrue(
-                $this->exists($path), "Unable to find a file or directory at path [{$path}]."
+                $this->exists($path),
+                "Unable to find a file or directory at path [{$path}]."
```
Comment (driesvints) [inline src/Illuminate/Filesystem/FilesystemAdapter.php:115]: Can you please undo all of these formatting changes?

**Review by driesvints — CHANGES_REQUESTED** [review]

File: src/Illuminate/Filesystem/FilesystemAdapter.php (hunk 1 above)
Comment (aneeskhan47) [reply src/Illuminate/Filesystem/FilesystemAdapter.php]: Sorry about that. it was my php-cs-fixer doing this. I undid all of the formatting changes 👍

**Comment by ankurk91** [comment]
We should wrap the api call in try/catch and all
`throw_if($this->throwsExceptions(), $e);`

**Pull request closed without merging**